web/posts/plasmapy-v01-release.rst

```diff
@@ -34,11 +34,11 @@
    through `all of our ~2000 commits <https://github.com/PlasmaPy/PlasmaPy/commits/master>`_,
    there is one thing that has nourished our souls, and pushed us to complete
    this release, and that is our `pretty plots
-   <http://docs.plasmapy.org/en/master/auto_examples/index.html>`_. Oh, and the
+   <http://docs.plasmapy.org/en/latest/auto_examples/index.html>`_. Oh, and the
```

Comment:
Actually I'm pretty darn sure that `latest` was not a thing back then, as it defaults to the last released version... but thanks for retroactively fixing this!

Reply:
Though this should probably be
```suggestion
   <http://docs.plasmapy.org/en/v0.1/auto_examples/index.html>`_. Oh, and the
```